tests/integration/records/single_db/test_records_load.py

```diff
@@ -56,9 +57,11 @@ def test_load_from_s3_records_directory(self):
                            "so skipping records directory URL test")
             return
 
-        if not self.has_scratch_bucket():
-            logger.warning('No scratch bucket, so skipping records directory URL test')
+        if not self.has_scratch_s3_bucket():
+            logger.warning('No scratch S3 bucket, so skipping records directory URL test')
             return
+        import os
+        print(f"XXX {os.environ['SCRATCH_S3_URL']}")
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker